package.json

```diff
@@ -23,18 +23,20 @@
     "doctor": "npm run check-node-version && ts-node bin/doctor",
     "check-node-version": "check-node-version --node \">= 12\"",
     "lint": "npm-run-all check-node-version lint:es lint:ts lint:sh lint:md",
-    "lint:es": "eslint --ignore-pattern node_modules/ --ignore-pattern fixtures/ \"{bin,examples,src,scripts,tests}/**/*.ts\"",
+    "test:lint": "npm-run-all check-node-version test:lint:es lint:ts lint:sh lint:md",
```

Comment:
Please keep the 'lint' prefix instead of changing to 'test' because all wechaty projects are following this name convention